### PR TITLE
chore(utxo-lib): improve readme of utxo-bin

### PR DESCRIPTION
**Pull request by rushilbg**
Add steps for upgrading the bin tool

BTC-0

### CODE REVIEW

**Review by saravanan7mani — DISMISSED** [review]

File: modules/utxo-bin/README.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ formatting them for terminal output.
 npm install --global @bitgo/utxo-bin
 ```
 
+# Upgrade
+```
+cd modules/utxo-bin
+yarn build
+```
+
```
Comment (OttoAllmendinger) [inline modules/utxo-bin/README.md:17]: I don't think we specific instructions for this module - the toplevel README should over it

doesn't `yarn install` on the toplevel work as well?

File: modules/utxo-bin/README.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ formatting them for terminal output.
...
+Either run using ``yarn ts-node bin/index.ts ...`` or install globally again using previous npm command.
```
Comment (OttoAllmendinger) [inline modules/utxo-bin/README.md:18]: the `yarn ts-node bin/index.ts` should be mentioned in a section `Usage` / `From Repo`

we should probably add a new section `Installation / From Repo` with `cd modules/utxo-bin; npm install --global`

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

File: modules/utxo-bin/README.md (hunk 1 above)
Comment (rushilbg) [reply modules/utxo-bin/README.md:17]: No it doesn't

Comment (OttoAllmendinger) [reply modules/utxo-bin/README.md:17]: Does `yarn build` on the toplevel do the right thing?

File: modules/utxo-bin/src/commands.ts [hunk 3]
```diff
@@ -183,7 +183,7 @@ export const cmdParseTx = {
         throw new Error(`conflicting arguments`);
       }
       console.log('Reading from stdin. Please paste hex-encoded transaction data.');
-      console.log('Press Ctrl-D to finish, or Ctrl-C to cancel.');
+      console.log('Press Ctrl-D to finish (after pasting tx hex), or Ctrl-C to cancel.');
```
Comment (OttoAllmendinger) [inline modules/utxo-bin/src/commands.ts:186]:
```suggestion
      console.log('After inserting data, press Ctrl-D to finish. Press Ctrl-C to cancel.');
```

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

File: modules/utxo-bin/README.md (hunk 1 above)
Comment (rushilbg) [reply modules/utxo-bin/README.md:17]: > error Command "build" not found.

Comment (OttoAllmendinger) [reply modules/utxo-bin/README.md:17]: The `yarn install` (or just `yarn`) command on the toplevel should build all packages. If it doesn't work that after that we have a bug that we need to fix separately.

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

**Review by OttoAllmendinger — APPROVED** [review]

**Pull request merged**